packages/govern/README.md

```diff
@@ -0,0 +1,90 @@
+# govern.js
+
+> The JavaScript client API for the Govern server
+
+## Usage
```

Comment:
Should we just point to the documentation website?

Reply:
Sounds good!